website/docs/docs/dbt-cloud-apis/project-state.md

```diff
@@ -14,7 +14,7 @@ There are two states that can be queried in dbt Cloud:
     
 - **Definition state** depends on what exists in the project given the code defined in it (for example, manifest state), which hasn’t necessarily been executed in the data platform (maybe just the result of `dbt compile`).
 
-### Definition (logical) vs. applied state of dbt nodes
+## Definition (logical) vs. applied state of dbt nodes
```

Comment:
fixing. per user docs best practices, avoid skipping heading levels (sequential is preferred)